src/lib/Builder.hs

```diff
@@ -1304,20 +1304,20 @@ getUnpacked atom = do
   atom' <- cheapNormalize atom
   ty <- getType atom'
   idxs <- projectionIndices ty
-  return $ idxs <&> flip getProjection atom'
+  forM idxs \is -> normalizeNaryProj is atom'
```

Comment:
Nit: Does this invalidate the comment above this function?

Reply:
I think it's still correct because `normalizeNaryProj` assumes that `atom` itself is already normalized.